client/src/components/sharedComponent/CircleProgressBar/index.js

```diff
@@ -29,29 +29,22 @@ class CircleProgressBar extends React.Component {
   render() {
     return (
       <div>
-        <BackButton />
+        <BackButton history={this.props.history} onClick={this.props.onClickBackButton} />
         <ProgressPage>
           <TreuIconBackground>
```

Comment:
there's a typo in this name.
```suggestion
          <TrueIconBackground>
```